### PR TITLE
Added appendix with DHS references and disclaimers

### PR DESCRIPTION
**Pull request by nissimsan**
Encouragement to consider if you should have your own contract numbers represented too. If so, please open issues to track for yourselves, or paste your contract numbers into the suggest changes feature of github.

### CODE REVIEW

**Review by OR13 — APPROVED** [review]

**Review by OR13 — CHANGES_REQUESTED** [review]
@nissimsan sorry to request changes on this, but there are a lot of white space changes... this should be contract numbers only.

**Comment by nissimsan** [comment]
@OR13, noted. Is this better?

**Review by OR13 — APPROVED** [review]

**Comment by OR13** [comment]
@tplooker you should add your contract numbers

File: index.html [hunk 1]
```diff
@@ -1426,5 +1426,16 @@ <h2>Security Considerations</h2>
 <div class="issue">TODO: We need to add a complete list of security
 considerations.</div>
     </section>
+    <section class="appendix">
+      <h2>Acknowledgements</h2>
+      <p>
+        Portions of the work on this specification have been funded by the
+        United States Department of Homeland Security's (US DHS) Silicon Valley
+        Innovation Program under contracts 70RSAT20T00000003, and
+        70RSAT20T00000033. The content of this specification does not
```
Comment (TallTed) [inline index.html:1435]: As elsewhere... Keeping all contract numbers in alphanumeric order, with one number per line in the HTML source, will make it easier to see whether a number has been added.
```suggestion
        Innovation Program under contracts 
        70RSAT20T00000003, 
        and
        70RSAT20T00000033. 
        The content of this specification does not
```

**Review by tplooker — APPROVED** [review]

**Comment by tplooker** [comment]
> @tplooker you should add your contract numbers

Yeap noted, will follow up in a subsequent PR

**Pull request merged**